bitnami/minio/templates/gateway/deployment.yaml

```
@@ -167,6 +167,44 @@ spec:
             - name: minio-console
               containerPort: {{ .Values.containerPorts.console }}
               protocol: TCP
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe:
```

If we apply this suggestion we could simplify the code
```suggestion
          livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.livenessProbe "enabled") "context" $) | nindent 12 }}
```

---

done